webpack/components/extensions/Hosts/BulkActions/BulkRepositorySetsWizard/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Modal, ModalVariant } from '@patternfly/react-core';
+import { useForemanModal } from 'foremanReact/components/ForemanModal/ForemanModalHooks';
+import BulkRepositorySetsWizard from './BulkRepositorySetsWizard';
+
+const BulkRepositorySetsWizardModal = () => {
+  const { modalOpen: isOpen } = useForemanModal({ id: 'bulk-repo-sets-wizard' });
+
+  return (
+    <Modal
+      width="60%"
```

Comment:
Please can it be 100% for people like me with small screens ( that need to debug)
![image](https://github.com/user-attachments/assets/3cc62401-c6e3-40c9-be68-f42e0385f7dd)

Reply:
> Please can it be 100% for people like me with small screens

At 100% it looks kinda crazy. All the other modals are 60% I think?

![image](https://github.com/user-attachments/assets/4588524f-89e6-4d43-b464-85b90a102caa)

Can we compromise and say 70%?
![image](https://github.com/user-attachments/assets/d1f27a9a-4b35-4806-b4df-9b1c78ee2a73)
![image](https://github.com/user-attachments/assets/587cc41d-6848-4b9b-988b-8619a2738e5c)